kafka-ui-api/src/test/java/com/provectus/kafka/ui/util/AccessControlServiceMock.java

```diff
@@ -16,7 +17,7 @@ public AccessControlService getMock() {
     when(mock.validateAccess(any())).thenReturn(Mono.empty());
     when(mock.isSchemaAccessible(anyString(), anyString())).thenReturn(Mono.just(true));
 
-    when(mock.isTopicAccessible(any(), anyString())).thenReturn(Mono.just(true));
+    when(mock.filterViewableTopics(any(), anyString())).thenReturn(Mono.just(Collections.emptyList()));
```

Comment:
I think it should be `when(mock.filterViewableTopics(any(), any())).then(i -> Mono.just(i.getArgument(0)));`

Reply:
thanks, done